src/neuroconv/tools/roiextractors/roiextractors.py

```diff
@@ -61,6 +63,45 @@ def get_default_ophys_metadata() -> dict:
         optical_channel=[default_optical_channel],
     )
 
+    metadata.update(
+        Ophys=dict(
+            Device=[default_device],
+            ImagingPlane=[default_imaging_plane],
+        ),
+    )
+
+    return metadata
+
+
+def get_default_imaging_metadata(
```

Comment:
or merge this into `get_nwb_imaging_metadata`

Reply:
Which do you prefer?

Reply:
OK, looks good!